modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -17,14 +19,19 @@ Updating or deleting Gateway API resources can result in downtime and loss of se
 * You have installed the OpenShift CLI (`oc`).
 * You have access to an {product-title} account with cluster administrator access.
 * Optional: You have backed up any necessary Gateway API objects.
++
+[WARNING]
+====
+Backup and restore can fail or result in data loss for any CRD fields that were present in the old definitions but are absent in the new definitions.
+====
 
 .Procedure
 
 . List all the Gateway API CRDs that you need to remove by running the following command:
 +
 [source,terminal]
 ----
-$ oc get crd | grep "gateway.networking"
+$ oc get crd | grep -F -e gateway.networking.k8s.io -e gateway.networking.x-k8s.io
```

Comment:
related to: https://github.com/openshift/openshift-docs/pull/93327#discussion_r2125081192